app/containers/ItunesContainer/selectors.js

```diff
@@ -0,0 +1,19 @@
+import { createSelector } from 'reselect';
+import { initialState } from './reducer';
+import get from 'lodash/get';
+/**
+ * Direct selector to the itunesContainer state domain
+ */
+
+export const selectItunesContainerDomain = (state) => state.itunesContainer || initialState;
+
+export const selectItunesContainer = () => createSelector(selectItunesContainerDomain, (substate) => substate);
+
+export const selectGridData = () =>
+  createSelector(selectItunesContainerDomain, (substate) => get(substate, 'gridData', null));
```

Comment:
let's not explicitly set the default value as null. It will be `undefined` anyway so let's leave it at that

Reply:
Noted

Reply:
Updated in the PR